plugins/mcreator-localization/help/default/biome/vines_block.md

```diff
@@ -1,3 +1,5 @@
 This paremeter controls the block used to replace vines.
 
-Select air block to don't have vines.
\ No newline at end of file
+Select air block to don't have vines.

```

Comment:
NOTE is no longer needed.

Also change 

Select air block to don't have vines.

to

Select air block to disable vines.

It is more english